set.mm

```diff
@@ -41910,13 +41925,20 @@ choice between (what we call) a "definitional form" where the shorter
 
   ${
     $d x A $.  $d x B $.
+    $( A variable introduction law for class equality, deduction version.
+       (Contributed by Thierry Arnoux, 2-Mar-2017.) $)
+    eqvincg $p |- ( A e. V -> ( A = B <-> E. x ( x = A /\ x = B ) ) ) $=
```

Comment:
This theorem is not a "deduction version" (of ~eqvinc?), since it is not of deduction form according our conventions (its conclusion does not have the antecedent ` ph -> `), but it is in "closed form" (using suffix "g"). ~eqvinc, however, is the inference version of ~eqvincg (see, for example, the comment of ~difexi).

Reply:
Agreed.
> deduction version --> closed form